packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx

```diff
@@ -81,21 +83,46 @@ export const DeckLayout = ({
     }
   }, [layoutParts]);
 
-  const searchEnabled = !!usePlugin('dxos.org/plugin/search');
-  const activeId = useMemo(() => Array.from(attention.attended ?? [])[0], [attention.attended]);
+  const [activeId, setActiveId] = useState<string | undefined>(Array.from(attention.attended ?? [])[0]);
+  const deckRef = useRef<HTMLDivElement>(null);
+  useLayoutEffect(() => {
+    let t: NodeJS.Timeout | undefined;
+    if (layoutMode === 'deck' && activeId) {
+      const el = deckRef.current?.querySelector(`article[data-attendable-id="${activeId}"]`);
+      // TODO(burdon): Generalize scroll into view by parent.
+      t = setTimeout(() => {
+        el?.scrollIntoView({ behavior: 'smooth', inline: 'center' });
+      }, 100); // TODO(burdon): Hack. The element already exists but scrolling doesn't happen without the delay. Calculate or memo offset?
+    }
+    return () => clearTimeout(t);
+  }, [layoutMode, activeId]);
```

Comment:
Maybe JSDoc this effect or move it into a separate hook file to declare/describe what it’s for.